back-end/src/models/users.js

```diff
@@ -13,14 +13,50 @@ async function getUserByEmail(email) {
 async function getUserByEmailAndPassword(email, pass) {
   const connection = await getConnection();
   const sql = `SELECT id, name, email, photo, level FROM user u WHERE  email = '${email}' AND password = '${pass}'; `;
-  const [ results ] = await connection.query(sql)
+  const [ results ] = await connection.query(sql);
 
   connection.end();
 
   return results[ 0 ];
 }
 
+async function listUserByNameForManager( name ) {
+  const connection = await getConnection();
+  const sql = `SELECT id, name, email, photo, level FROM user u WHERE name='${ name }';`;
+  const [ results ] = await connection.query( sql );
+
+  connection.end();
+
+  console.log( results );
+
+  return results[ 0 ];
+}
+
+async function addUser( name, pass, email, photo, level, isActive ) {
```

Comment:
Seguindo a sugestão do outro comentário, aqui ficaria um pouco diferente também
```suggestion
async function addUser( user ) {
const { name, pass, email, photo, level, isActive } = user;
```

Reply:
Aceito